DBMS/DBMS-Assignments/DBMS-Assignment-2/Assignment_2.sql

```diff
@@ -0,0 +1,205 @@
+(3,4,1,(select price from product where product_id=4)*1,'2019-08-03 12:20:36',null) ,
+(4,9,2,(select price from product where product_id=9)*2,'2019-08-03 13:20:36',null) ,
+(5,10,1,(select price from product where product_id=10)*1,'2019-08-03 14:20:36',null) ,
+(5,12,1,(select price from product where product_id=12)*1,'2019-08-04 08:20:36',null) ,
+(6,1,2,(select price from product where product_id=1)*2,'2019-08-04 10:20:36',null) ,
+(2,13,3,(select price from product where product_id=13)*3,'2019-08-04 11:20:36',null) ,
+(7,14,4,(select price from product where product_id=14)*4,'2019-08-05 19:20:36', null) ;
+
+/*
+SET FOREIGN_KEY_CHECKS = 0;
+TRUNCATE TABLE order_details;
+*/
+
+-- insert into order_details
+
+INSERT INTO order_details(cart_id,address,date_ordered,total_price,order_status) VALUES
+(2,'Udaipur','2019-08-01 09:20:36',(select sum(price) from cart where cart_id=2 and item_status is null GROUP BY cart_id ),'placed'),
+(3,'delhi','2019-08-05 10:20:36',(select sum(price) from cart where cart_id=3 and item_status is null GROUP BY cart_id),'placed') ,
+(5,'bikaner','2019-08-06 08:20:36',(select sum(price) from cart  where cart_id=5 and item_status is null GROUP BY cart_id),'placed'),
+(6,'jaipur','2019-08-06 11:20:36',(select sum(price) from cart  where cart_id=6 and item_status is null GROUP BY cart_id),'placed'),
+(7,'udaipur','2019-08-06 12:20:36',(select sum(price) from cart  where cart_id=7 and item_status is null GROUP BY cart_id),'placed');
+
+-- update status of cart
+
+UPDATE cart SET item_status='delivered' WHERE cart_id=2 AND product_id=2 ;
+UPDATE cart SET item_status='shipped' WHERE cart_id=2 AND product_id=3 ;
+UPDATE cart SET item_status='placed' WHERE cart_id=3 AND product_id=5 ; 
+UPDATE cart SET item_status='shipped' WHERE cart_id=3 AND product_id=1 ;
+UPDATE cart SET item_status='returned' WHERE cart_id=3 AND product_id=4 ;
+UPDATE cart SET item_status='delivered' WHERE cart_id=5 AND product_id=10 ;
+UPDATE cart SET item_status='delivered' WHERE cart_id=5 AND product_id=12 ;
+UPDATE cart SET item_status='placed' WHERE cart_id=6 AND product_id=1 ;
+UPDATE cart SET item_status='delivered' WHERE cart_id=2 AND product_id=13 ;
+UPDATE cart SET item_status='cancelled' WHERE cart_id=7 AND product_id=14 ;
+
+/*
+2.b Display Id, Title, Category Title, Price of the products which are Active and 
+	recently added products should be at top. 
+*/
+SELECT product_id, product_name, price ,category_name FROM product 
+INNER JOIN  category on
+product.category_id=category.category_id
+ORDER BY date_added DESC; 
+
+
+-- 2.c Display the list of products which don't have any images.
+
+SELECT product_id ,product_name FROM product WHERE product_id NOT IN 
+(SELECT product_id FROM images);
+
+
+/*
+2.d Display all Id, Title and Parent Category Title for all the Categories listed, sorted by 
+	Parent Category Title and then Category Title.
+*/
+SELECT category_id ,category_name ,ifnull(parent_category,'TOP CATEGORY') AS parent
```

Comment:
`IFNULL` should be capital and the column name should be `Parent Category` not `Parent`